src/main/scala/rocket/PTW.scala

```diff
@@ -26,11 +26,15 @@ class PTWReq(implicit p: Parameters) extends CoreBundle()(p) {
 class PTWResp(implicit p: Parameters) extends CoreBundle()(p) {
   val ae_ptw = Bool()
   val ae_final = Bool()
+  /**page fault  */
   val pf = Bool()
+  /**guest page fault  */
   val gf = Bool()
+  //hypervisor wrx
   val hr = Bool()
   val hw = Bool()
   val hx = Bool()
+  /**  return the target PTE */
```

Comment:
return to which component? where is the signal source, where are the signal sinks? This documentation doesn't give any additional information than the signal name: `PTE`.
```suggestion
  /** return the target PTE */
```